src/customers/controller/customer.controller.ts

```diff
@@ -0,0 +1,56 @@
+import { Body, Controller, Get, Post, Param } from '@nestjs/common';
+import { CustomerService } from '../service/customer.service';
+import { CreateCustomerDTO } from '../dto/create-customer.dto';
+import { Customer } from '../domain/customer.domain';
+import { CustomerMapper } from '../mapper/customer.mapper';
+import { CustomerDTO } from '../dto/customer.dto';
+import { ApiResponse } from '@nestjs/swagger';
+
+@Controller('customers')
+export class CustomerController {
+  private readonly customerMapper: CustomerMapper;
```

Comment:
I think we can use something like this:

```
@Controller('customers')
export class CustomerController {
  constructor(
    private readonly customerService: CustomerService,
    private readonly customerMapper: CustomerMapper,
  ) {}
}
```

Reply:
Actually, make the methods `static` and use them without injecting anything in the `Controller`.
Also the naming here can be adjusted to be better.
We should have something la this:
- CustomerMapper.toDTO()
- CustomerMapper.toEntity()
- CustomerMapper.toCreateDTO()